examples/with_pyspark/with_pyspark/repository.py

```diff
@@ -51,5 +51,5 @@ def make_and_filter_data():
 
 
 @repository
```

Comment:
apologies if this was already discussed in the project layout thread, but in a project like this that has only ops are we not recommending an `ops` folder (like the `assets` folder)

Reply:
np at all! i dont think we've discussed it in practical cases like tihs.

imo the benefit of having an assets folder is that we can load assets from modules and assets can show up as they are newly defined, whereas for ops/jobs/graphs, it'd be a bit tricky to draw the line for which layer should be in its own file - so since this DAG is small, i decided to keep it simple, in a single.